src/content/docs/accounts/accounts/account-maintenance/promotions-ui.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Product promotions 
+tags:
+  - Accounts
+  - Accounts and billing
+  - General account settings
+metaDescription: 'How to sign up for product promotions'
+freshnessValidatedDate: 2024-07-15 
+---
+
+
+import accountsPreviewsandTrials from 'images/accounts_screenshot-crop_previews-and-trials.webp'
+
+import accountsPromotionsTable from 'images/accounts_screenshot-crop_promotions-table.webp'
+
+import accountsManagePreview from 'images/accounts_screenshot-crop_manage-preview.webp'
+
+import accountsPromoUpgradeButton from 'images/accounts_screenshot-crop_promo-upgrade-button.webp'
+
+
+
+Promotions are special offers to try out product features at New Relic. You can browse through our various promotions and enroll through our promotions management UI. 
+
+The type of promotion currently available is the product preview. This is the opportunity to try out a feature before it is released (pre-GA). The promotions management UI gives you:
+
+* One-stop shopping for previews : View all available product previews in one place
+* Enrollment flexibility: Easily sign-up or leave from product previews until they become generally available (GA)
+* Upgrade options: Upgrade to access products once they are generally available 
+
+To help you get started, we've provided some tips about eligibility and enrollment.
+
+## Eligibility [#eligibility]
+
+You can check the plan management page to see if your organization is eligible for a product promotion. If you don't see the promotions UI, your organization isn't eligible.
```

Comment:
I would put info (or a link to the section below) here on how to see the promotions UI